packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -234,6 +237,26 @@ export class NextjsExportPodUtils {
     out.stdout?.pipe(process.stdout);
     return out.pid;
   }
+
+  static async loadSidebarsFile(
+    sidebarFilePath: string | false | undefined | null
+  ): Promise<unknown> {
+    if (sidebarFilePath === false) {
+      return DisabledSidebar;
+    }
+
+    if (_.isNil(sidebarFilePath)) {
+      return DefaultSidebar;
+    }
+
+    // Non-existent sidebars file: no sidebars
+    if (!(await fs.pathExists(sidebarFilePath))) {
```

Comment:
should throw an error here telling user we were unable to find it

Reply:
[`3111e6d` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/3111e6d99239a6bcf16529865cf0a566c5ac9f47)

Reply:
can we also add a test for this in [[../packages/engine-test-utils/src/__tests__/pods-core/NextjsExportPod.spec.ts]]?

Reply:
[`73d7552` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/73d755279fe32277bf6f103905ace0d3f46da723)